cmd/main.go

```diff
@@ -88,3 +98,32 @@ func bindKeysToEnvAndDefault() {
 		viper.SetDefault(b.Key, b.Default)
 	}
 }
+
+func exitListener() {
+	signalChanel := make(chan os.Signal, 1)
+	signal.Notify(signalChanel,
+		syscall.SIGTERM)
+	go signalHandler(signalChanel)
+}
+
+func signalHandler(signalChanel chan os.Signal) {
+	kicsRunArgs := []string{
+		killCommand,
+		viper.GetString(params.KicsContainerNameKey),
```

Comment:
@tiagobcx  Here we could check if the  container id exists and only then try to cancel it.